source/server-discovery-and-monitoring/tests/unified/logging-replicaset.yml

```diff
@@ -0,0 +1,269 @@
+description: "replicaset-logging"
+
+schemaVersion: "1.14"
+
+runOnRequirements:
+  - topologies:
+      - replicaset
+    minServerVersion: "4.4" # awaitable hello
+
+createEntities:
+  - client:
+      id: &setupClient setupClient
+      useMultipleMongoses: false
```

Comment:
nit: I think we can omit this option since it's a replica set; it won't have any effect.